test/fixtures/parser/postgres/postgres_single_quote.sql

```diff
@@ -0,0 +1,15 @@
+SELECT '';
+
+SELECT '''';
+
+SELECT '
+
+';
+
+SELECT '''aaa''';
+
+SELECT '
+''
+';
+
+SELECT '\';
```

Comment:
Can you fix the newlines:

![image](https://user-images.githubusercontent.com/10931297/133481442-84a1b23e-c9cd-422d-95b4-d7e64264584f.png)

```suggestion
SELECT '\';

```

Not least of all cause it would make my editorconfig change easier 😁